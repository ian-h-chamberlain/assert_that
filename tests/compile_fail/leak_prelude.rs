extern crate assert_that;

use assert_that::assert_that;

fn main() {
    assert_that!("Hello World", str::similar("Hello World"));

    assert!(always());
}
