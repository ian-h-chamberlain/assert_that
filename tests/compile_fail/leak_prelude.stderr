error[E0425]: cannot find function `always` in this scope
 --> $DIR/leak_prelude.rs:8:13
  |
8 |     assert!(always());
  |             ^^^^^^ not found in this scope
  |
help: possible candidates are found in other modules, you can import them into scope
  |
3 | use predicates::constant::always;
  |
3 | use predicates::prelude::predicate::always;
  |
